### PR TITLE
[Feature] - Added Cards component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named alerts under which markup has been written in `cards.html` and the stylesheet file is written in `cards.css`. The stylesheet file `cards.css` only consists of the styles related to buttons i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main styles.css file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds a Card component of different types.

### CODE REVIEW

**Review by yashpatel1797 — COMMENTED** [review]
The code looks good but there are 5 cards in P0 so you can add that designs.

File: cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  margin: 1rem;
+}
+
+.card-img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 30rem;
+  height: 15rem;
+  max-width: 100%;
```
Comment (poojavishnoi) [inline cards/cards.css:27]: Do use object-fit:cover for images, It's a good practise.Also I don't think flex is needed here

File: cards/cards.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <title>FoundOve UI</title>
+  </head>
+  <body>
+    <div class="container">
```
Comment (poojavishnoi) [inline cards/cards.html:13]: Use more specific class name like card-container.

**Comment by parthpandyappp** [comment]
> The code looks good but there are 5 cards in P0 so you can add that designs.

Got it! Will add more.

File: cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
+.disp-card {
```
Comment (TejasShekar) [inline cards/cards.css:1]: I feel a better class name would be needed for this

**Comment by TejasShekar** [comment]
Components looks great. But I would suggest you to use better class names as menu-opt and menu-opt-container for Read and Bookmark doesn't feel like an ideal class name.  Maybe use card-options and card-options-list.

**Review by TejasShekar — COMMENTED** [review]
Components looks great. But I would suggest you to use better class names as menu-opt and menu-opt-container for Read and Bookmark doesn't feel like an ideal class name.  Maybe use card-options and card-options-list.

File: cards/cards.css (hunk 3 above)
Comment (parthpandyappp) [reply cards/cards.css:1]: Actually, I've used a common naming style for displaying items as `disp-componentName`. Anyways, I respect your strong opinion.

**Comment by parthpandyappp** [comment]
> Components looks great. But I would suggest you to use better class names as menu-opt and menu-opt-container for Read and Bookmark doesn't feel like an ideal class name. Maybe use card-options and card-options-list.

Cool, feels like that! I'll change it.

File: cards/cards.html (hunk 2 above)
Comment (parthpandyappp) [reply cards/cards.html:13]: Actually, I've not made any specific container for holding components in it. So, I've considered a generalized `container`  class which is coming from the root style.

File: cards/cards.css (hunk 1 above)
Comment (parthpandyappp) [reply cards/cards.css:27]: Sure, Got it. Will change it. Thankyou :smile:

Comment (parthpandyappp) [reply cards/cards.css:27]: @poojavishnoi Anyways, I'm doing `object-fit: cover` here as I missed on it but due to some reason that is blurring the image a bit may be due to forceful fitting of the image in the container. Any solution for that?

**Comment by parthpandyappp** [comment]
I've added a few more cards. Also, take a note, I've defined a temporary `chip` class in the `cards.css` file so as to generate an appropriate deploy preview at this time. Later, while merging the changes, I'll remove its the definition from the `cards.css` file as the main class comes from `chips.css` from the `Chips` component once it gets merged.

**Review by Logan1x — APPROVED** [review]

**Pull request merged**